source/tests/system/nimxlcterminaladaptor_restricted_driver_api_tests.cpp

```diff
@@ -48,20 +48,15 @@ class NiMxLcTerminalAdaptorRestrictedDriverApiTests : public Test {
   std::unique_ptr<NimxlcTerminalAdaptorRestricted::Stub> nimxlcterminaladaptor_stub_;
 };
 
-nidevice_grpc::Session init_session(const client::StubPtr& stub, const std::string& hostname)
+nidevice_grpc::Session init_session(const client::StubPtr& stub)
 {
-  auto response = client::create_session(stub, hostname.c_str());
+  auto response = client::create_session(stub, "");
   EXPECT_EQ(0, response.status());
   EXPECT_EQ(0, response.c_status().code());
   auto session = response.handle();
   return session;
 }
 
-nidevice_grpc::Session init_session(const client::StubPtr& stub)
-{
-  return init_session(stub, DEFAULT_HOSTNAME);
```

Comment:
It seems that the `DEFAULT_HOSTNAME` constant is never used and can be removed (or should be changed to `""` and used in the new code?).

Reply:
Thanks for catching that! I've removed the constant.